scripts/execute_ebpf_cicd_stress_tests.ps1

```diff
@@ -0,0 +1,36 @@
+﻿# Copyright (c) Microsoft Corporation
+# SPDX-License-Identifier: MIT
+
+param ([parameter(Mandatory=$false)][string] $AdminTarget = "TEST_VM",
+       [parameter(Mandatory=$false)][string] $StandardUserTarget = "TEST_VM_STANDARD",
+       [parameter(Mandatory=$false)][string] $LogFileName = "TestLog.log",
+       [parameter(Mandatory=$false)][string] $WorkingDirectory = $pwd.ToString(),
+       [parameter(Mandatory=$false)][string] $TestExecutionJsonFileName = "test_execution.json",
+       [parameter(Mandatory=$false)][bool] $Coverage = $false,
+       [parameter(Mandatory=$false)][string] $SelfHostedRunnerName,
+       [parameter(Mandatory=$false)][string] $RestartExtension)
+
+Push-Location $WorkingDirectory
+
+$AdminTestVMCredential = Get-StoredCredential -Target $AdminTarget -ErrorAction Stop
+$StandardUserTestVMCredential = Get-StoredCredential -Target $StandardUserTarget -ErrorAction Stop
+
+# Load other utility modules.
+Import-Module .\common.psm1 -Force -ArgumentList ($LogFileName) -WarningAction SilentlyContinue
+Import-Module .\vm_run_tests.psm1  -Force -ArgumentList ($AdminTestVMCredential.UserName, $AdminTestVMCredential.Password, $StandardUserTestVMCredential.UserName, $StandardUserTestVMCredential.Password, $WorkingDirectory, $LogFileName) -WarningAction SilentlyContinue
+
+# Read the test execution json.
+$Config = Get-Content ("{0}\{1}" -f $PSScriptRoot, $TestExecutionJsonFileName) | ConvertFrom-Json
+$VMList = $Config.VMMap.$SelfHostedRunnerName
+
+# Run stress tests on test VMs.
+foreach ($VM in $VMList) {
+    Invoke-CICDStressTestsOnVM -VMName $VM.Name -Coverage $Coverage -RestartExtension $RestartExtension
```

Comment:
you're already doing this, but just calling out that these tests need to be deployed and run in a VM - otherwise you can crash the runner machine, and break all the other runner instances executing on the same host in parallel.

Reply:
Umm, that _was_ the whole point of the `Invoke-CICDStressTestsOnVM` :-) Unless I'm misunderstanding your comment?